cmd/gen-go-sample/main.go

```diff
@@ -362,17 +362,17 @@ func (g *generator) genSample(sampConf schema_v1p2.Sample, methConf GAPICMethod)
 	p("// [START %s]", sampConf.RegionTag)
 	p("")
 
-  // comments
-  emptyArr := make([]string, 0)
+  // generate comments above the sample function
+  var emptyArr []string = nil
   requiresNewLine := false
   
   if sampConf.Description != "" {
-  	writeComment(sampConf.Description, emptyArr, g)
+  	writeComment(strings.TrimSuffix(sampConf.Description, "\n"), emptyArr, g)
```

Comment:
I might just use [`strings.TrimSpace`](https://golang.org/pkg/strings/#TrimSpace) to normalize the comment, removing leading & trailing white space. WDYT? Is leading white space necessary?

Reply:
Done.